static/app/views/alerts/rules/issue/details/ruleDetails.tsx

```diff
@@ -56,9 +62,12 @@ class AlertRuleDetails extends AsyncComponent<Props, State> {
       organization,
       rule_id: parseInt(params.ruleId, 10),
     });
+    if (this.state.rule) {
+      findIncompatibleRules(this.setIncompatible, this.state.rule);
+    }
```

Comment:
This is in both `componentDidMount` and `componentDidUpdate` right now because in the tests, the state is already loaded by the time didmount is called